448. Find All Numbers Disappeared in an Array.py

```diff
@@ -0,0 +1,14 @@
+class Solution:
+    def findDisappearedNumbers(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: List[int]
+        """
+        output = []
+        numbers = set(nums)
+        
+        for i in range(1, len(nums) + 1):
```

Comment:
this can be simplified into:
`return [i for i in range(1, len(nums) + 1) if i not in numbers]`